### PR TITLE
Close multiple positions

### PR DESCRIPTION
**Pull request by alexxyu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,19 +289,19 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
...
-| [Labcorp](https://careers.labcorp.com/global/en/job/COVAGLOBAL2254695EXTERNALENGLOBAL/Software-Developer-internship) | Durham, NC; or Indianapolis, IN | Software Developer Internship (2023 Global Internship Program) |
+| Labcorp | Durham, NC; or Indianapolis, IN | **🔒 Closed 🔒** Software Developer Internship (2023 Global Internship Program) |
 | [Sierra Nevada Corporation](https://snc.wd1.myworkdayjobs.com/en-US/SNC_External_Career_Site/job/Englewood-CO/Software-Engineer-Intern--Summer-2023-_R0018868) | Englewood, CO; or Huntsville, AL; or Hagerstown, MD | Software Engineer Intern (Summer 2023) |
 | [Sysco](https://careers.sysco.com/jobs/R79428?lang=en-us) | Houston, TX; or Austin, TX | Intern - Software Development Engineer (10 Week during Summer)（undergrad） |
 | DoorDash | Multiple Locations | Software Engineer Intern: [US Position](https://boards.greenhouse.io/doordash/jobs/4560561?gh_jid=4560561), [Toronto Position](https://boards.greenhouse.io/doordash/jobs/4560283?gh_jid=4560283) |
-| [United](https://careers.united.com/us/en/job/WHQ00020911/Intern-Digital-Technology-Summer-2023) | Chicago, IL | Digital Technology Intern (no sponsorship) |
```
Comment (ewang2002) [inline README.md:302]: United seems to still be open.

image

I was able to apply to it just now. What does it show on your side?

**Review by ewang2002 — CHANGES_REQUESTED** [review]
Hey, thanks for essentially cleaning up a bunch of entries!

United seems to still be open on my side; I was able to apply to it minutes ago. What does it show on your side when you try to apply? 

If someone else can confirm that United is closed, I can merge this PR. Otherwise, it's probably best to revert this specific change. Your call, though.

File: README.md (hunk 1 above)
Comment (alexxyu) [reply README.md:302]: Ah, it took me to a "listing not found" page yesterday, but I'm able to see the posting now too. I'll update the PR.

**Review by ewang2002 — APPROVED** [review]
Awesome, looks great! And yeah, the whole "listing is down today but up the next day" happens a lot for some odd reason -- guess companies can't make up their minds.

Thanks again! :)

**Pull request merged**